main.py

```diff
@@ -1,25 +1,35 @@
 from forum_service import ForumService
-from post_cleaner import PostCleaner
 from db_manager import DBManager
+from Models.forum_post_model import ForumPostModel
 
+#initialize our classes
 db = DBManager()
+fs = ForumService()
 
-# url1 = "http://services.runescape.com/m=forum/forums.ws?17,18,812,66119561,goto,{}"
-# url2 = "https://services.runescape.com/m=forum/sl=0/forums?17,18,769,66133050,goto,{}"
-# db.track_new_thread(url1)
-# db.track_new_thread(url2)
+#track new threads
+url1 = "http://services.runescape.com/m=forum/forums.ws?17,18,812,66119561,goto,{}"
+url2 = "https://services.runescape.com/m=forum/sl=0/forums?17,18,769,66133050,goto,{}"
+db.track_new_thread(url1)
+db.track_new_thread(url2)
 
+#lets try pulling down all the info in both threads
 active_threads = db.fetch_active_threads()
+for url, last_page_num, last_post_num, thread_id in active_threads:
+   data = fs.get_clean_forum_posts(url, thread_id, 1)
+   
+   #this will create a list of ForumPost objects using the properly cleaned forum post information
+   forum_posts = []
+   for post in data:
+      forum_posts.append(ForumPostModel.from_array(post))
```

Comment:
converting these from a list to a `ForumPostModel` object should occur in `PostCleaner` -- see my comments in post_cleaner.py